### DISCUSSION TITLE
zh/guide/drivers/baidu.photo

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#96] [Announcements]
# zh/guide/drivers/baidu.photo

刷新令牌（同百度网盘的一样） 点此 (https://openapi.baidu.com/oauth/2.0/authorize?responsetype=code&clientid=iYCeC9g08h5vuP9UqvPHKKSVrKFXGa1v&redirect_uri=https://tool.nn.ci/baidu/callback&scope...

https://alist.nn.ci/zh/guide/drivers/baidu.photo.html

**Comment by lstr2** [comment]
访问 https://photo.baidu.com/youai/album/v1/list. 获取{tid}时，搜索id（021858707431029901）无法搜索到。

**Comment by lstr2** [comment]
访问 https://photo.baidu.com/youai/album/v1/list. 获取{tid}时，搜索id（4021858707431029901）无法搜索到

**Comment by luoyang871111** [comment]
配置完成后，一直打不开，转圈圈

**Reply by GemkingYum** [reply]
我也是 一直打不开，console里报超时

**Comment by enccvv** [comment]
启用签名有什么作用？

**Comment by 235buluo** [comment]
有的Tid搜索不到，请问是怎么回事，是不是获取链接出问题了

**Comment by Lparksi** [comment]
请问 token 的有效期大概有多长时间

**Reply by simplove** [reply]
我也想知道这个问题

**Comment by edward0726** [comment]
为啥会出现“下载失败。权限不足。”的问题

**Comment by huangda1982** [comment]
通过Alist网页可以上传，但是WebDav不能上传。

**Reply by blackness3** [reply]
是的 ，我也发现有这个问题，无法通过webdav上传。

**Comment by RainoutOvO** [comment]
Cannot read properties of undefined (reading 'status')

**Comment by dunjiaow** [comment]
An error occurred: API rate limit exceeded for alist-org/docs. Sign in to increase the rate limit请问这是怎么了

**Comment by zhlx2835** [comment]
failed link: failed get link: errno: 9019, refer to https://photo.baidu.com/union/doc
这什么错

**Reply by soest0201** [reply]
我也遇到了 感觉是下载接口出了问题

**Comment by maghaoye** [comment]
failed link: failed get link: errno: 9019, refer to https://photo.baidu.com/union/doc
这是什么情况

**Comment by IxxiaobaiI** [comment]
您好，以前还是好好的，最近不知为何点击一刻相册分享链接提示
{
    "code": 500,
    "message": "failed link: failed get link: errno: 9019, refer to https://photo.baidu.com/union/doc",
    "data": null
}
本人实在没办法，希望寻求各位大佬或者作者大大的帮助

**Reply by soest0201** [reply]
用不了了 换地址存吧

**Comment by hjs523** [comment]
现在更新新版本后点击进去提示这个：failed link: failed get link: not found 302 redirect

**Comment by Ci9Quester** [comment]
有没有人在alist store里也是只能加载缩略图，点开大图是fail to load content，并且通过raidrive挂载到本地后能显示目录结构和文件属性但是全部无法预览并且点开也死活加载不出来图片的？
